### ISSUE TITLE
Allow to set up initial target route in UI Unit Test

### ISSUE THREAD

**Issue by mcollovati** [vaadin/testbench#1759]
Currently, a UI Unit Test always navigates to the main route (blank path), then in the test the developer can navigate to another view before starting assertions.

It would be nice to have a way to instruct the test to directly navigate to a target root, without the need to instantiate the main route.
